### ISSUE TITLE
mWeb - Rate per mile updated is not sync in real time on second login

### ISSUE THREAD

**Issue by kbecciv** [Expensify/App#23173]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Issue found when executing PR**  https://github.com/Expensify/App/pull/22243

## Action Performed:
1. Go to Reimbursement page and click on first section, you will get a popup page.
2. Check by changing the data in the second section (Track Distance) for the rate and unit, so that it gets reflected after saving on the reimbursement page.
3. Open the 3rd section (Unlock next-day reimbursements), and check see if its opening correctly.
4. Go offline and then turn online, and see if the component gets sync.
5. Open the app in a different place (anywhere...desktop, mobile) and change the data in second section (Track Distance)

## Expected Result:
The rate updated must sync in real time on second device

## Actual Result:
The rate updated not sync in real time on second device (mWeb and iOS)

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android / native
- [ ] Android / Chrome
- [ ] iOS / native
- [x] iOS / Safari
- [ ] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:** 1.3.42-21
**Reproducible in staging?:** y
**Reproducible in production?:** y
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** Any additional supporting documentation

**Expensify/Expensify Issue URL:**

https://github.com/Expensify/App/assets/93399543/e785f3ee-6edb-49e2-b5d7-08d7dc34a232

**Issue reported by:** Applause- Interna Team
**Slack conversation:**

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by mateocole** [comment]
hey @marcochavezf wanted to get a double check here. I dont think this is a bug since it synced upon refresh. I am not sure if it is possible to sync this any more real time, or if we want to optimize for that. Let me know what you think

**Comment by KrAbhas** [comment]
I tried to recreate the situation on local as well as stage but for me the rendering is pretty fast as compared to the added video. 

I have checked with a combination on all six platforms (i.e; iOS, Android, web, desktop, mobile-chrome, mobile-safari)

There may be certain points that can be considered like network latency or battery optimisation of device that maybe putting one of the process in waiting queue (better if we can test on 2 devices rather than 2 windows - still on my Mac the 2 windows situation also works perfectly fine)

**Comment by marcochavezf** [comment]
Hi @mateocole, in theory we should send a pusher event to update the account in real-time, so I think this is a bug 
@kbecciv just to confirm, is this only happening on the second login? (i.e. the rate per mile is sync with the primary login?). Also is this only happening on Safari?

**Comment by kbecciv** [comment]
Hi @marcochavezf I tried with combination Android app and Android/Chrome, but not able to reproduce it. 

https://github.com/Expensify/App/assets/93399543/828df651-3d71-4171-aa33-92e8bd76e0e4

I looked dipper for this issue, posting comments and video.
Combinations: 
1. If primary login IOS app changed the rate -  secondary login mWeb/safari is failed (not updated)
2. If secondary login mWeb/safari changed the rate -  primary login IOS app is Pass 9 rate updated)


https://github.com/Expensify/App/assets/93399543/f8760143-fde7-48d2-b947-9489d82ae72c
